Unable to open BirthdayStatistics

Error msg:
Caused by: java.lang.NullPointerException
	at address.controller.BirthdayStatisticsController.setPersonData(BirthdayStatisticsController.java:54)
	at address.controller.MainController.showBirthdayStatistics(MainController.java:316)
	at address.controller.RootLayoutController.handleShowBirthdayStatistics(RootLayoutController.java:211)
	... 56 more

--- a/src/main/java/address/controller/BirthdayStatisticsController.java
+++ b/src/main/java/address/controller/BirthdayStatisticsController.java
@@ -51,6 +51,9 @@ public class BirthdayStatisticsController {
         // Count the number of people having their birthday in a specific month.
         int[] monthCounter = new int[12];
         for (Person p : persons) {
+            if (p.getBirthday() == null){
+                continue;
+            }
             int month = p.getBirthday().getMonthValue() - 1;
             monthCounter[month]++;
         }

